slice/internal/controller/workload_controller.go

```diff
@@ -256,22 +256,13 @@ func (r *WorkloadReconciler) cleanupSlices(ctx context.Context, wl *kueue.Worklo
 		return false, err
 	}
 
-	deleted, deformed, errored, other := r.groupSlices(slices)
+	deleted, errored, other := r.groupSlices(slices)
```

Comment:
```suggestion
	deleted, toDelete, other := r.groupSlices(slices)
```
Should it be toDelete?